lib/cli.go

```diff
@@ -81,6 +81,7 @@ func run(c *cli.Context) error {
 
 	for _, name := range names {
 		fn := gapi.RegisteredGAPIs[name]
+
```

Comment:
what's this, and the one below for?

Reply:
Oops sorry, didn't properly clean up after having some debug output here.

Fixed.